client/actions/main.ts

```diff
@@ -452,16 +452,31 @@ const post = (original, updates = {}, withConfirmation = true) => (
         return promise
             .then(
                 (rtn) => {
-                    if (!confirmation && rtn) {
+                    let failedPlanningIds, item;
+
+                    if (Array.isArray(rtn) && rtn.length === 2) {
+                        [item, {failedPlanningIds}] = rtn;
+                    } else {
+                        item = rtn;
+                        failedPlanningIds = [];
+                    }
+
+                    if (!confirmation && item) {
                         notify.success(
                             gettext(
                                 'The {{ itemType }} has been posted',
                                 {itemType: typeString}
                             )
                         );
                     }
+                    failedPlanningIds?.map((failedItem) => notifyError(
+                        notify,
+                        failedItem,
+                        gettext(
+                            'Related item {{ itemId }} Post Validation Failed',
```

Comment:
not sure if the id in message is useful for the user, might be better to just say that some related items failed validation and make it somehow visible in the UI which ones those were

Reply:
This default message remains hidden from users under normal conditions. It will only be displayed if there's a scenario outside the usual conditions :- https://github.com/superdesk/superdesk-planning/blob/d3d0eb0cc496d12d9475c95317deb396dc6ed5c1/client/utils/index.ts#L250-L276